src/fio/fioEngine.js

```diff
@@ -269,6 +269,31 @@ export class FioEngine extends CurrencyEngine {
             }
             return res
           }
+          case 'addBundledTransactions': {
+            const res = await this.multicastServers(actionName, params)
+            const fioAddress = this.otherData.fioAddresses.find(
+              ({ name }) => name === params.fioAddress
+            )
+
+            if (!fioAddress) return res
```

Comment:
put this before multicast and of false - throw

Reply:
Also, don't test for "null" value implicitly.
If you want to check if something is null or undefined, use the correct:
`            if (fioAddress == nulll) return res`